package.json

```diff
@@ -11,15 +11,20 @@
   "scripts": {
     "start": "node app.js",
     "pretest": "npm run lint",
-    "test": "npm run test:unit && npm run test:e2e",
-    "test:e2e": "echo 'no e2e tests available.'",
+    "test": "npm run test:unit",
+    "test:e2e": "npm run selenium:install && npm run selenium:start && npm run nightwatch",
```

Comment:
Are you saying that if you just run test:e2e, that will fail?  Because I don't like that; that is the suck and I should fix that before checking in.  Because those are in there.